src/main/resources/+matlab/+ciplugins/JenkinsLoggingPlugin.m

```diff
@@ -0,0 +1,36 @@
+classdef JenkinsLoggingPlugin < matlab.buildtool.plugins.BuildRunnerPlugin
```

Comment:
This looks great! I learned after I made the plugin example that the namespace I used was not so great. `matlab.ciplugins` implies that we are shipping the ciplugins package with the matlab product. Maybe we should call the namespace something like `jenkins.plugins`

Reply:
`jenkins.plugin` sounds bit intervening to me. is it Jenkins plugin or Matlab plugin my personal favorite is `matlab.plugins`

Reply:
This file will be copied just after run build is complete and then run build itself invoke action followed by copy. I see less or no chances that any one could delete this file programmatically or otherwise. since this file comes from MATLAB code it does not have direct access to root folder. through MATLAB i could copy the file either in workspace somewhere or in temp directory. With temp directory deletion of file manually is possible by some othr process.

Reply:
The `matlab` root namespace is generally reserved for stuff that ships with MATLAB. Perhaps just `jenkins`?

Reply:
or something like the following?
`ciplugins.jenkins` <-------- I think my favorite.
`ciintegrations.jenkins`
`ci.jenkins`

PRISM standards suggest that the first package is the name of the "product" or I would say "offering". Our product here may be our `ciplugins`

Reply:
`ciplugins.jenkins` would be a good option